kalite/topic_tools/__init__.py

```diff
@@ -1,618 +0,0 @@
-"""
-TODO: NOTHING SHOULD BE HERE! It's prohibiting the import of other topic_tools.xxx
```

Comment:
Huge step! Wonderful!! I'm sure from the experience of the problems with topic_tools.settings that this can be a benefit for the rest of ka-lite's life cycle, no matter which kinds of refactors we get into.